packages/app/tests/utils/formatters.spec.ts

```diff
@@ -44,7 +44,7 @@ describe("formatters:", () => {
     );
   });
   it("returns formatted Hex data", () => {
-    expect(formatHexDecimals("32770", "Hex")).toBe("0x032770");
+    expect(formatHexDecimals("32770", "Hex")).toBe("0x32770");
```

Comment:
Was the formatting incorrect before?

Reply:
It's the same value as leading zeros can be ignored. Explained [here](https://github.com/matter-labs/block-explorer/pull/299#discussion_r1819368551).